backend/typescript/services/implementations/reviewService.ts

```diff
@@ -498,18 +499,38 @@ class ReviewService implements IReviewService {
     return result;
   }
 
-  async getReviews(): Promise<ReviewResponseDTO[]> {
-    let reviews: PgReview[];
-    let result: ReviewResponseDTO[];
+  getPaginationOffset(page: string, limit: number): number {
+    const offset = page ? parseInt(page, 10) * limit : 0;
+    return offset;
+  }
+
+  async getReviews(
+    page: string,
+    size: string,
+  ): Promise<RetrieveReviewResponseDTO> {
+    let result: RetrieveReviewResponseDTO;
 
     try {
       result = await this.db.transaction(async (t) => {
-        reviews = await PgReview.findAll({
+        const limit = size ? parseInt(size, 10) : 10;
+        const offset = this.getPaginationOffset(page, limit);
+
+        const { rows, count } = await PgReview.findAndCountAll({
           transaction: t,
           include: [{ all: true, nested: true }],
+          limit,
+          offset,
         });
 
-        return reviews.map((r) => ReviewService.pgReviewToRet(r));
+        const currentPage = page ? parseInt(page, 10) : 0;
+        const totalPages = Math.ceil(count / limit);
+
+        return {
+          totalReviews: count,
```

Comment:
This count seems to be wrong for some reason, and it's throwing off the `totalPages` as well. For example, I only have 5 reviews here in my database but it says I have 11:
![image](https://user-images.githubusercontent.com/19916973/180103618-d0783b77-3e6e-4032-933e-52afe5eb0d47.png)

Reply:
Wait that's weird the count seems to be working for me. @Saber74 Can you also try and confirm if the count is off

Reply:
@chriistine I think this might be counting books instead of reviews for some reason, but I'm not too sure :thinking:. Try it with the seed data to recreate. There should be only 7 reviews, but `totalReviews` is 14 for me.